src/jailer/src/env.rs

```diff
@@ -462,10 +462,9 @@ impl Env {
 
         // We do a copy instead of a hard-link for 2 reasons
         // 1. hard-linking is not possible if the file is in another device
-        // 2. while hardlinking would save up disk space and also memory by
-        //    sharing parts of the Firecracker binary (like the executable .text
-        //    section), this latter part is not desirable in Firecracker's
-        //    threat model. Copying prevents 2 Firecracker processes from
+        // 2. while hardlinking would save up disk space and also memory by sharing parts of the
```

Comment:
Nit: this commit title should be `style: cargo fmt`.

Reply:
Updated.